src/web/event-notifications/event-notification-types/SlackNotificationSummary.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import CommonNotificationSummary from 'components/event-notifications/event-notification-types/CommonNotificationSummary';
+
+class SlackNotificationSummary extends React.Component {
```

Comment:
This should be a function component instead of class - https://reactjs.org/docs/components-and-props.html#function-and-class-components